logging-metrics-tracing/logging.md

```diff
@@ -83,4 +103,22 @@ For instance:
           "syslog": false,
           "stdout": true
       }
-    }
\ No newline at end of file
+    }
+
+# Logstash
+If you want to log using the Logstash standard via stdout, you have to add the `krakend-logstash` integration in the
+root level of your `krakend.json`, inside the `extra_config` section. **The `gologging` needs to be enabled too**.
+
+For instance:
+
+    "extra_config": {
+      "github_com/devopsfaith/krakend-logstash": {
+        "enabled": true
+      }
+      "github_com/devopsfaith/krakend-gologging": {
```

Comment:
This example should have the `"format": "logstash"` defined.

Reply:
Added! :)

Reply:
Thanks @rgomezcasas , documentation is live now:
https://www.krakend.io/docs/logging-metrics-tracing/logging/#logstash